### PR TITLE
Update mailmap.

### PR DESCRIPTION
**Pull request by silene**

### CODE REVIEW

**Review by Zimmi48 — APPROVED** [review]

**Comment by Zimmi48** [comment]
@silene I'll leave it to you to put the appropriate milestone before I merge.

**Comment by silene** [comment]
Since I will do at least one more pipeline run when backporting #14855, I might just as well backport this one along the way.

**Comment by Zimmi48** [comment]
@coqbot merge now

**Pull request merged**
